src/søknad/steg/3-barnadine/LeggTilBarn.tsx

```diff
@@ -37,12 +63,12 @@ const LeggTilBarn: React.FC<Props> = ({ settÅpenModal }) => {
 
   const tilbakestillFelt = () => {
     settBarnDato(dagensDato);
-    settNavn('Barn');
+    settNavn('');
     settPersonnummer('');
     settBoHosDeg('');
   };
 
-  const leggTilBarn = () => {
+  const leggTilBarn = (id: string | undefined) => {
```

Comment:
Hvorfor er id av typen `string `eller `undefined`? Og i hvilke tilfeller vil den være `undefined`?

Reply:
Det er bare når man endrer barn at leggTilBarn tar in `id`. Når man legger til et barn, så sender man ikke med id, siden den opprettes inni leggTilBarn-funksjonen. Kunne jo også ha lagd to funksjoner, én for å legge til og én for å endre, tror du det hadde vært bedre?